cmd/mailbox-controller/main.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
+
+	kcpclient "github.com/kcp-dev/kcp/pkg/client/clientset/versioned"
+	kcpinformers "github.com/kcp-dev/kcp/pkg/client/informers/externalversions"
+
+	edgeindexers "github.com/kcp-dev/edge-mc/pkg/indexers"
+)
+
+func main() {
+	resyncPeriod := time.Duration(0)
```

Comment:
Why resyncPeriod is 0 ? usually default is a large value (e.g. 10 * time.Hour )

Reply:
I write controllers that do not need any resync at all.

Reply:
Does `resyncPeriod = 0` mean no resync ? Is this documented somewhere ?

Reply:
Sadly the generated informer code lacks good comments.  It is based on the generic library, which documents the answer to your question.  See https://github.com/kubernetes/client-go/blob/v0.24.9/tools/cache/shared_informer.go#L214-L224